katalog/nfs-subdir-external-provisioner/MAINTENANCE.md

```diff
@@ -0,0 +1,45 @@
+# `nfs-subdir-external-provisioner` Package Maintenance
+
+To prepare a new release of this package:
+
+1. Generate manifests from helm charts
+```bash
+helm repo add nfs-subdir-external-provisioner https://kubernetes-sigs.github.io/nfs-subdir-external-provisioner
+helm template --release-name nfs-subdir-external-provisioner nfs-subdir-external-provisioner/nfs-subdir-external-provisioner --namespace nfs-subdir-external-provisioner | tee deploy.yaml
```

Comment:
can we write down here the version of the chart that you used? so the next maintainer knows where to start from.

Reply:
I edit the file specifing the current `nfs-subdir-external-provisioner` helm chart version.
Morover I moved the bash in the file `MAINTENANCE.sh`